arch/inst/Zicsr/csrrw.yaml

```
@@ -13,7 +13,7 @@ description: |
   If `xd=x0`, then the instruction shall not read the CSR and shall not
   cause any of the side effects that might occur on a CSR read.
 definedBy: Zicsr
-assembly: xd, imm, xs1
+assembly: xd, csr, imm
```

`csrrw` writes a register value to the desired CSR, not an immediate value.
```suggestion
assembly: xd, csr, xs1
```